functions/package.json

```diff
@@ -19,7 +19,7 @@
     "debug": "^4.1.1",
     "express-async-errors": "^3.1.1",
     "firebase-admin": "^8.12.1",
-    "firebase-functions": "^3.7.0",
+    "firebase-functions": "3.6.1",
```

Comment:
Is there any specific reason we are using "firebase-functions": "3.6.1" and not 3.7.2?

Reply:
"firebase serve" doesnt work in 3.7.x

Reply:
firebase serve has never worked for me. Does it really works with 3.6.1?

Reply:
yes